src/utils/static-cache.json

```diff
@@ -0,0 +1,456 @@
+{
+  "npm_react": "https://github.com/facebook/react/",
```

Comment:
@stefanbuck I think this is causing the e2e tests to fail in the octolinker repo. They expect a different url than the one you used here https://github.com/OctoLinker/OctoLinker/blob/main/e2e/fixtures/javascript/index.js#L10

Reply:
Thanks 🙏 I'll submit a fix soon